### PR TITLE
Add card ui

### PR DESCRIPTION
**Pull request by bambong**
Add Card UI and Connect with Script

### CODE REVIEW

File: Assets/Scripts/Cards/Card.cs [hunk 1]
```diff
@@ -80,20 +100,6 @@ public void SetOff()
 
     public virtual void CardEffect() {}
```
Comment (nick-choi) [inline Assets/Scripts/Cards/Card.cs:101]: You always try to put similar methods together. 

You should move these lines(101, 102) after SetDescriptions().
They are virtual methods and can be implemented by subclasses.

File: Assets/Scripts/Cards/Card.cs [hunk 2]
```diff
@@ -47,19 +52,34 @@ public class Card : MonoBehaviour
...
+    
+
+
     private void Start()
     {
         rectTransform = GetComponent<RectTransform>();
+        
+
     }
     public RectTransform GetRect() => rectTransform;
```
Comment (nick-choi) [inline Assets/Scripts/Cards/Card.cs:71]: You always try to put similar methods together.

Move this line after GetUseType().
They just return some values.
Generally, programmers call them setter/getter.

File: Assets/Scripts/Cards/Card.cs (hunk 1 above)
Comment (bambong) [reply Assets/Scripts/Cards/Card.cs:89]: https://github.com/SuperRookies/Slay24/pull/14/commits/4e7c0310302d207b22b8bac2114b1a53707894cc

File: Assets/Scripts/Cards/Card.cs (hunk 2 above)
Comment (bambong) [reply Assets/Scripts/Cards/Card.cs]: https://github.com/SuperRookies/Slay24/pull/14/commits/32d10ba587981dd08755cfb3dcc16ce45f95e33c

Comment (bambong) [reply Assets/Scripts/Cards/Card.cs]: it look better than before, Thanks

**Comment by nick-choi** [comment]
I suggest you guys check Format before committing like below.
visual_studio_code_format

**Pull request merged**